Containerfile

```diff
@@ -1,7 +1,7 @@
 # Copyright Contributors to the Packit project.
 # SPDX-License-Identifier: MIT
 
-ARG base_image=centos:8
+ARG base_image=quay.io/app-sre/centos:8
```

Comment:
are we sure this image is going to be up to date?

Reply:
[The repo](https://quay.io/repository/app-sre/centos?tab=info) says "Mirror of the upstream centos repository" and it was updated a week ago so "hopefully" is the only answer I have.